ksql-engine/src/test/java/io/confluent/ksql/QueryTranslationTest.java

```diff
@@ -104,23 +105,50 @@ public void shouldBuildAndExecuteQueries() {
   }
 
   @Parameterized.Parameters(name = "{0}")
-  public static Collection<Object[]> data() throws IOException {
-    final String topologyVersion = System.getProperty(TOPOLOGY_VERSION_PROP, CURRENT_TOPOLOGY_VERSION);
-    final String topologyDirectory = TOPOLOGY_CHECKS_DIR + "/" + topologyVersion;
-    final Map<String, TopologyAndConfigs> expectedTopologies = loadExpectedTopologies(topologyDirectory);
+  public static Collection<Object[]> data() {
+    final String[] topologyVersions =
+        System.getProperty(TOPOLOGY_VERSIONS_PROP, CURRENT_TOPOLOGY_VERSIONS)
+            .split(TOPOLOGY_VERSIONS_DELIMITER);
```

Comment:
Can we make the default behavior to scan `TOPOLOGY_CHECKS_DIR` and run against all the versions found there? That way, when we add a new version we just have to add the directory.

Reply:
Done.